### ISSUE TITLE
Meta-Issue: Audio Derivatives

### ISSUE THREAD

**Issue by dannylamb** [Islandora-CLAW/CLAW#930]
This is a meta-issue to track development of audio derivative generation. Please refer to this issue in any subsequent issues to link them.

Currently, Islandora CLAW can handle audio objects, and you can migrate existing audio derivatives from 7.x into CLAW, but you can't generate them on a green field installation.

We should start with our usual microservice approach, that is

- Develop a HTTP wrapper around lame as a microservice
- Emit events from Drupal to an activemq queue
- Read events from the queue using a camel listener issue a request to the microservice, storing the results in the repository.

**Comment by Natkeeran** [comment]
@dannylamb 

Islandora 7.x supports mp3 audio derivatives.  [aac](https://en.wikipedia.org/wiki/Advanced_Audio_Coding)/[he-aac](https://en.wikipedia.org/wiki/High-Efficiency_Advanced_Audio_Coding) seems to be the successor to mp3.  Though aac has a slightly smaller lower support from devices, it has better quality and size.  

While keeping the mp3 option as configurable, should we consider providing aac audio derivatives.  

7.x vagrant does use `libfaac` to create audio tracks for video derivatives.  Thus, ffmpeg can be used for aac derivatives for audio.

Also, ffmpeg supports creation of mp3 via libmp3lame.  Can we use this instead of lame.

**Comment by dannylamb** [comment]
@Natkeeran Yes absolutely.  I'd love to see it all wrapped up under ffmpeg using Homarus.
